english/avatar/which-one.md

```diff
@@ -0,0 +1,6 @@
+You're the Avatar? Cool!
+
+Which one are you?
+
+[Aang]()
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.